Rakefile

```diff
@@ -5,6 +5,8 @@ require_relative 'config/application'
 
 Rails.application.load_tasks
 
-RuboCop::RakeTask.new do |task|
-  task.requires << 'rubocop-rails'
-end
+###===== Wasn't running with this but it was in the docs
+
+# RuboCop::RakeTask.new do |task|
+#   task.requires << 'rubocop-rails'
+# end
```

Comment:
Strange this was causing the issue. As you said the docs recommend to add this code block but perhaps we placed it in the wrong spot?